src/popup-prompt/lionel.js

```diff
@@ -182,9 +207,9 @@ const getPrice = (countryCode, withDiscount) => {
 const getSubscriptionPromptValues = (countryCode, withDiscount) => {
 	const price = getPrice(countryCode, withDiscount);
 	if (countryCode === 'USA' || withDiscount) {
```

Comment:
Whats `withDiscount`?

Reply:
With discount was something previously added when there used to be a flag called priceFlashSale but that flag no longer exists. @alexnaish reckons it was left in the code in case we needed to use that flag again.

Reply:
I guess we shouldn't show a different offer ID and message if we were to resurrect the message though